packages/ag-charts-community/src/chart/interaction/keyNavManager.ts

```diff
@@ -108,6 +116,11 @@ export class KeyNavManager extends BaseManager<KeyNavEventType, KeyNavEvent> {
             case 'Space':
             case 'Enter':
                 return this.dispatch('submit', 0, event);
+            case 'Escape':
+                return this.dispatch('cancel', 0, event);
+            case 'Backspace':
+            case 'Delete':
+                return this.dispatch('delete', 0, event);
```

Comment:
Is the `KeyNavManager` the correct place for these events? They are not specifically about navigation, but may affect it. And the abstraction between key code and event is good.

Reply:
This is a good place for this, because this is where we convert key-codes to actions. I don't think that it will have a negative effect on navigation, because most listeners will only care about blur, focus, tab, nav-hori and nav-vert.

It does feel like the class could have a better name than "`KeyNavManager`" but we can live with that.

Reply:
I think so, the responsibility of this class is to abstract keyboard interactions, these seem to fit within that scope.